lib/pbench/test/unit/server/query_apis/test_datasets_detail.py

```diff
@@ -138,8 +138,16 @@ def test_bad_dates(self, client, server_config, user_ok, pbench_token):
     @pytest.mark.parametrize(
         "user", ("drb", "", "no_user", None),
     )
+    @pytest.mark.parametrize("authentication", (True, False))
```

Comment:
Shouldn't there three cases, not two:  valid authentication/authorization, invalid auth, and missing auth?
(We need to chat about the difference between "authentication" and "authorization"....)